packages/storybook-vue/src/stories/Link/Link.stories.mdx

```diff
@@ -0,0 +1,187 @@
+import { ArgsTable, Canvas, Meta, Story } from "@storybook/addon-docs";
+import { Link } from "@utrecht/component-library-vue";
+import { argsTypes, componentArgs } from "./argTypes.ts";
+
+<Meta id="vue-link" title="Vue.js Component/Link" component={Link} argsTypes={argsTypes} />
+
+# Link
+
+export const Template = (args) => ({
+  components: { Link },
+  setup() {
+    return { args };
+  },
+  template: `<div dir="${args.dir}" lang="${args.lang}"><Link v-bind="args">${args.default}</Link></div>`,
+});
```

Comment:
All these stories also render a `default` attribute in HTML, as well as `dir` and `lang` attributes. Is this a bug?

<img width="883" alt="Screenshot 2022-07-26 at 10 46 01" src="https://user-images.githubusercontent.com/30694/180964436-7841e033-d3aa-4f48-ab35-fe140a8df274.png">

Reply:
Yes, I think this is a bug, here is the reference https://github.com/storybookjs/storybook/discussions/12691
I tried a couple of scenarios but it is still the same, also it's because these properties are not exist in the component itself we tried to add them through the storybook.
But, I don't see it as a big issue because it's only happening in storybook